src/Microsoft.AspNet.Mvc.Core/Rendering/HtmlHelperOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.Rendering
+{
+    /// <summary>
+    /// Provides programmatic configuration for the HTML helpers and <see cref="ViewContext"/>.
+    /// </summary>
+    public class HtmlHelperOptions
+    {
+        public HtmlHelperOptions()
+        {
+            IdAttributeDotReplacement = "_";
```

Comment:
You could initialize these values inline 
`public string IdAttributeDotReplacement { get; set; } = "_";` to avoid having to document this constructor.